motivaa-backend/motivaa-app/src/test/java/com/motivaa/TestUtils/HabitFactory.java

```diff
@@ -0,0 +1,83 @@
+package com.motivaa.TestUtils;
+
+import com.motivaa.entity.DaySpecificHabit;
+import com.motivaa.entity.Habit;
+import com.motivaa.entity.NotDaySpecificHabit;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class HabitFactory {
+    public static String VALID_USER_UUID = "123e4567-e89b-12d3-a456-426614174000";
+    public static String VALID_NAME = "HabitName";
+    public static String VALID_RECURRING_TYPE_SPECIFIC_DAY = "SPECIFIC_DAY";
+    public static String VALID_RECURRING_TYPE_NON_SPECIFIC_DAY = "NON_SPECIFIC_DAY";
+    public static String VALID_LIST_OF_RECURRING_DAYS = "MONDAY;TUESDAY;WEDNESDAY";
```

Comment:
This is unrelated to this change but I am wondering now why are you handling the recurring days like this. Why won't the frontend send it as a valid json array in the first place so you don't have to deal with this string split all the time?

Reply:
I am not a frontend guru but I think it even takes extra effort on the frontend to concatonate it to this format instead of sending it a json array.

Reply:
its a fair point, I think its a bad call due to lack of experience. I can imagine us changing it later on. :)